src/pretix/base/models/items.py

```diff
@@ -501,6 +501,17 @@ class Item(LoggedModel):
                     "in which both products are visible while all tickets in the referenced quota are reserved, "
                     "but not yet sold.")
     )
+    hidden_if_item_available = models.ForeignKey(
+        'Item',
+        null=True, blank=True,
+        on_delete=models.SET_NULL,
+        verbose_name=_("Only show after sellout of"),
```

Comment:
For old events using the quota-setting at least the german translation fails when using the same string in english. Not sure it is worth the effort though as `hidden_if_available` is anyways deprecated.

Reply:
I don't like the generic German wording and I don't have a better one, so I've added a context parameter